cmd/kyma/alpha/deploy/opts.go

```diff
@@ -64,16 +64,20 @@ func (o *Options) validateTimeout() error {
 // validateFilters sets up all filters that will be used by kustomize when running the command
 func (o *Options) validateFilters() error {
 	var filters []kio.Filter
-	modifier, err := kustomize.LifecycleManagerImageModifier(
-		o.LifecycleManager,
-		func(image string) {
-			o.NewStep(fmt.Sprintf("Used Lifecycle-Manager: %s", image)).Success()
-		},
-	)
-	if err != nil {
-		return err
+
+	if o.LifecycleManager != "" {
+		modifier, err := kustomize.LifecycleManagerImageModifier(
+			o.LifecycleManager,
+			func(image string) {
+				o.NewStep(fmt.Sprintf("Used Lifecycle-Manager: %s", image)).Success()
```

Comment:
This Used `"Used Lifecycle-Manager: %s"`is still worth printing out even for the default case to indicate user which image of Lifecycle Manager get deployed.